packages/client/src/runtime/utils/getClientEngineType.ts

```diff
@@ -0,0 +1,35 @@
+import { GeneratorConfig } from '@prisma/generator-helper'
+
+export enum ClientEngineType {
+  NodeAPI = 'node-api',
+  Binary = 'binary',
+}
+export const DEFAULT_CLIENT_ENGINE_TYPE = ClientEngineType.Binary
+
+export function getClientEngineType(
+  generator?: GeneratorConfig,
```

Comment:
```suggestion
  generatorConfig?: GeneratorConfig,
```
Just a thought to make it easier to know what it expects